### ISSUE TITLE
Accessibility Review of our Style Guide

### ISSUE THREAD

**Issue by mrmetaverse** [omigroup/website#39]
Let's conduct an Accessibility Review of our [Style Guide](https://github.com/omigroup/website/issues/29)

_Originally posted by @mrmetaverse in https://github.com/omigroup/website/issues/29#issuecomment-953101801_

**Comment by robertlong** [comment]
I can help with this 👍

**Comment by jsejms** [comment]
I can help as well, already posted some quick suggestions, changing over font weights, color contrasts, and white font colors on CTA's from the black on purple we had.

What is Shadow's handle on Github?  I dont see them to add.
@mrmetaverse 

https://www.figma.com/file/7IkelFTsQPpMwzrWGzqFBA/Omigroup-Designs?node-id=676%3A529

**Comment by mrmetaverse** [comment]
@jsejms and @Elirudite Hey there! We are doing backlog refinement and I wanted to shake this up. How do we feel about completing this next? @technobaboo we would benefit from your XR Accessibility eye as well.

**Comment by technobaboo** [comment]
I'll def take a look then!

**Comment by Elirudite** [comment]
I'm definitely down for this! Whenever you all are ready.

**Comment by mrmetaverse** [comment]
bump 

Lets do a formal pass, and document what we do as an accessibility review. Then move this to done, with the understanding that we should always review for accessibility 

@Elirudite @technobaboo @jsejms

**Comment by Elirudite** [comment]
When/how would you like to do this?

**Comment by jsejms** [comment]
I've got time next Tuesday after 6pm EST.

**Comment by mrmetaverse** [comment]
Ping me when you feel it's "done" or just move it into "In Progress" and then back to "Review" when done if you prefer. :) 

I won't be around next Tuesday (I am teaching a class) but will keep my eye out and am at your service otherwise.

**Comment by Elirudite** [comment]
@jsejms did an accessibility review today. Here are our notes:

- Skipping Header sizes (not a huge problem, but it can get problematic once we add to the website)
- - Subheader (Header is h1, subheader is h3)
- - Core Value headers (Section header is an h2, core values are h5 and h6)
![image](https://user-images.githubusercontent.com/10410478/161866731-3d5bbc69-e3c1-42fb-921e-65ae207a214a.png)
![image](https://user-images.githubusercontent.com/10410478/161866740-42d8c5fe-7b3b-46fe-81fa-5f4ba7ff4f83.png)

-  Primary CTA button needs better contrast (can pass with a larger text and a darker purple)

- The big OMI logo behind the core values is blurry, this is not an accessibility issue, but would be cleaner if it was uploaded as an SVG. Can also either be smaller, or appear on top of the section backgrounds above and below, similar to how it is on figma.

- The ‘How you can get involved’ section does not read the BG colors. It's being read as white on white for some reason, may need to check how wordpress is handling the background image/gradient.
- - The hover states for the buttons are purple on purple, may need to consider a different color

Overall, our site is mostly accessible according to AA standards!! Just need to clean up those colors and text sizes to ensure everything passes.

(Feel free to update if I missed anything)
